combat-trainer.lic

```diff
@@ -2259,16 +2261,42 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.aiming_trainables
+        skill = game_state.determine_aiming_skill
+
+        if skill.eql?('Brawling')
+          3.times{ return if !execute_aiming_action?('gouge', game_state) }
```

Comment:
💡 There might be some room to re-use some code between Brawling and Tactics. After all, this line is the same as `%w[gouge gouge gouge]`

Reply:
Or there might be a more general pattern here. The skill determines the array of actions, and you can make `wield_offhand_weapon` and `sheath_offhand_weapon` a little more defensive then always call them, even for Brawling/Tactics.

I suspect this block would get somewhat smaller, and the pattern would be more apparent. I don't know if it would be any clearer overall though.

Reply:
I thought of doing this, with something like whirlwind_trainables, etc. I'll look more into it.

Reply:
Ok, handled this.